client.py

```diff
@@ -92,7 +92,7 @@ def main():
                     logger.info('reached limit')
                     break
                 scheduler.submit(s)
```

Comment:
What happens in the case that `i + s["count"] > limit`?  I think we need to modify s["count"] before submitting.